### DISCUSSION TITLE
Testnet status

### DISCUSSION THREAD

**Discussion by aug2uag** [gochain-io/docs#130] [Q&A] [answered]
is testnet down? I've been trying to run a script, erroring and found that the last transaction on testnet seemed to be a couple days ago

would you please verify status
<img width="1176" alt="Screen Shot 2022-02-02 at 3 28 20 PM" src="https://user-images.githubusercontent.com/1901195/152255093-1a13d148-69e8-4b38-ab98-fc986e50d559.png">

**Comment by treeder** [comment]
We're looking into it. cc @r-gochain

**Comment by r-gochain** [comment, marked as answer]
Resolved
